braid/braid.h

```diff
@@ -1217,6 +1253,51 @@ braid_SetRichardsonEstimation(braid_Core core,                /**< braid_Core (_
                               braid_Int  local_order          /**< Local order of the time integration scheme, e.g., local _order=2 for backward Euler */
                               );
 
+/** 
+ * Turn on low-rank Delta correction. This uses Jacobians of the fine-grid time-stepper as a linear correction to the coarse time-stepper. 
+ * This can potentially greatly accelerate convergence for nonlinear systems.
+ * 
+ * The action of the Jacobian will be computed on a (low-rank) time-dependent basis initialized by the user.
+ * TODO: Finish this entry
+ */
+braid_Int
+braid_SetDeltaCorrection(braid_Core           core,        /**< braid_Core (_braid_Core) struct */
+                         braid_Int            rank,        /**< Integer, sets number of Lyapunov vectors to store */
+                         braid_PtFcnInitBasis basis_init,  /**< Function pointer to routine for initializing basis vectors */
+                         braid_PtFcnInnerProd inner_prod   /**< Function pointer to routine for computing inner product between two vectors (needed for Gram-Schmidt orthonormalization) */
+                         );
+
+
+/**
+ * Defer the low-rank Delta correction to a coarse level or to a later iteration. 
+ * To mitigate some of the cost of Delta correction, it may be turned off on the first few fine-grids,
+ * or turned off for the first few iterations.
+ * 
+ */
+braid_Int
+braid_SetDeferDelta(braid_Core core,   /**< braid_Core (_braid_Core) struct*/
+                    braid_Int  level,  /**< Integer, Delta correction will be deferred to this level (Default 1)*/
```

Comment:
So, this skips delta correction on the fine level, and just starts it on the first coarse level, right?